python/lsst/qa/explorer/functors.py

```diff
@@ -767,3 +793,169 @@ class Seeing(Functor):
     def _func(self, df):
         return 0.168*2.35*np.sqrt(0.5*(df['ext_shapeHSM_HsmPsfMoments_xx']**2 +
                                        df['ext_shapeHSM_HsmPsfMoments_yy']**2))
+
+
+class e1(Functor):
+    name = "Ellipticity e1"
```

Comment:
Should further document this as a "distortion" ellipticity; there are a few different conventions, and this is as good as any, but while "ellipticity" is ambiguous, "distortion" is not.